docs/assembly.rst

```diff
@@ -805,7 +838,7 @@ Grammar::
     IdentifierOrList = Identifier | '(' IdentifierList ')'
     IdentifierList = Identifier ( ',' Identifier)*
     AssemblyAssignment = '=:' Identifier
-    LabelDefinition = Identifier ( '[' ( IdentifierList | NumberLiteral ) ']' )? ':'
+    LabelDefinition = Identifier ( '[' ( IdentifierList? | NumberLiteral ) ']' )? ':'
```

Comment:
The new `?` seems to allow `label[]:` (which I guess is the same as `label:`).  I'm wondering if this is somehow independent of the rest of the PR or an integrated part of the PR.